test/Generator/MethodGeneratorTest.php

```diff
@@ -365,6 +366,8 @@ public function returnTypeHintClassesProvider()
             [NullableReturnTypeHintedClass::class, 'otherClassReturn', '?\\' . InternalHintsClass::class],
             [IterableHintsClass::class, 'iterableReturnValue', 'iterable'],
             [IterableHintsClass::class, 'nullableIterableReturnValue', '?iterable'],
+            [ObjectHintsClass::class, 'objectReturnValue', 'object'],
```

Comment:
These should only be added to the data providrr if the php version is 70200 or greater